src/appshell/qml/DevTools/KeyNav/KeyNavExample.qml

```diff
@@ -1,62 +1,155 @@
 import QtQuick 2.15
 import MuseScore.Ui 1.0
+import QtQuick.Layouts 1.15
 
 Rectangle {
     id: root
 
     color: ui.theme.backgroundSecondaryColor
 
-    Column {
-        anchors.fill: parent
+    property string lastClickedInfo: ""
 
-        KeyNavSection {
-            anchors.left: parent.left
-            anchors.right: parent.right
-            color: "#fce94f"
+    Rectangle {
+        id: infoPanel
 
-            sectionName: "sect 2"
-            sectionOrder: 2
-            subsecCount: 3
+        anchors.left: parent.left
+        anchors.right: parent.right
+        height: 64
+
+        Text {
```

Comment:
Please, use StyledTextLabel here (even in devtools)

Reply:
fixed